content/acra/security-controls/client-id-sql-detection/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: ClientID SQL Detection
+weight: 12
+---
+
+## ClientID Detection from SQL
+
+{{< hint info >}}
+This feature is available in [Acra Enterprise Edition](/acra/enterprise-edition/) only.
+{{< /hint>}}
+
+AcraServer supports the ability to retrieve the ClientID from SQL in the runtime dynamically.
+
+AcraServer uses a default behavior in which the ClientID is retrieved from TLS
+certificates [`--tls_client_id_from_cert` flag]({{< ref "acra/configuring-maintaining/general-configuration/acra-server/#tls" >}}), provided by the client during handshake.
```

Comment:
you know, that you can write simply `[--tls_client_id_from_cert flag](/acra/configuring-maintaining/general-configuration/acra-server/#tls)`, right? :)